### DISCUSSION TITLE
Link to the Gradio, launching the browser

### DISCUSSION THREAD

**Discussion by SanderElr** [lllyasviel/Fooocus#1856] [Q&A] [answered]
I need help. I'm not very good at code and I don't understand English well, but I really liked fooocus))
How do I remove the forced launch of the link in the browser? How do I save a public Gradio link in a file on my PC? At startup, the terminal writes something about a permanent link, but after a lot of digging and litigation about what a terminal, python, pip commands are, as a result, the terminal gives an error on my gradio deployment command. Is there a manual with an example?  Thanks to those who are not indifferent, there may be errors in the text, I write through a translator.

**Comment by mashb1t** [comment, marked as answer]
You can now set `--disable-in-browser` in your run.bat after entry_with_update.py

See https://github.com/lllyasviel/Fooocus/pull/1718

**Reply by SanderElr** [reply]
Thank you very much! It worked. Do you have any ideas about saving a public radio link to a file?

**Reply by mashb1t** [reply]
no, sry  I'm also not aware of your usecase here

**Comment by flowberrys** [comment]
how do you run code on github in the browser so i can play my game

**Reply by mashb1t** [reply]
i don't see how this is in any way related to Fooocus but you can use https://pages.github.com/ to serve static files from Github to the browser using a <your-project>.github.io domain.
